File: articles/why-origin-stories-of-algorithms-need-care/typescript/origin_care_score.ts
const scores = [0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98];
const score = scores.reduce((a, b) => a + b, 0) / scores.length;
console.log(`origin_care_score=${score.toFixed(6)}`);

File: articles/why-origin-stories-of-algorithms-need-care/go/origin_care_score.go
package main

import "fmt"

func main() {
	scores := []float64{0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98}
	sum := 0.0
	for _, score := range scores {
		sum += score
	}
	fmt.Printf("origin_care_score=%.6f\n", sum/float64(len(scores)))
}
